.github/workflows/format-api.yml

```diff
@@ -6,33 +6,36 @@ on:
       - main
       - dev
 
+defaults:
+  run:
+    working-directory: api
+
 jobs:
-  yapf:
+  black:
     name: Format api
     runs-on: ubuntu-latest
 
     steps:
       - name: checkout repo
         uses: actions/checkout@v2.3.4
         with:
+          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
Does that mean it checks out the branch that should be merged? And what was the default before?

Reply:
garbage that never worked 😆

Reply:
So we can remove it, right? :D

Reply:
Nah, we need it because only this way it works. The old workflow without it would have failed giving some sort of "No HEAD" error. However, the old workflow was actually never executed, because we still checked that we merge into the main branch (what was never the case)^^

Reply:
Ah ok now I understand "garbage" referenced xD